weather.py

```diff
@@ -24,7 +24,12 @@ def convert_date(iso_string):
     Returns:
         A date formatted like: Weekday Date Month Year e.g. Tuesday 06 July 2021
     """
-    pass
+    from datetime import datetime
```

Comment:
`datetime` has already been imported in Line 2 of `weather.py`, so no need to import it again here.
We usually import all our external modules at the very top of the file, so they can be accessed by all of the functions in that file.